setup.py

```diff
@@ -54,14 +75,15 @@ def get_versions():
              # former naming schema?
              "-mt"
            ]
-for suf in suffixes:
-    lib = find_library("boost_python%s" % suf)
-    if lib is not None:
-        libs.append("boost_python%s" % suf)
-        break
-else:
-    raise Exception("Cannot find boost_python library")
-
+    for suf in suffixes:
+        print("finding boost_python%s" % suf )
+        lib = find_library("boost_python%s" % suf)
+        print(lib)
+        if lib is not None:
+           libs.append("boost_python%s" % suf)
+           break
+        else:
```

Comment:
Sorry, I missed that that's wrong, too. That `else:` branch actually belongs to the for loop (i.e. remove one level of indention).